<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipes</class>
 <widget class="QMainWindow" name="Recipes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>691</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>808</width>
    <height>691</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>808</width>
    <height>691</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recipes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="recipe_list_tableview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>641</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionAdd_Recipe"/>
    <addaction name="actionRemove_Recipe"/>
   </widget>
   <addaction name="menuManage"/>
  </widget>
  <action name="actionAdd_Recipe">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove_Recipe">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
